src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -351,6 +338,73 @@ private SmugglerProgressBase.Counts ProcessIndexes(SmugglerResult result)
             return result.Indexes;
         }
 
+        private void WriteIndex(SmugglerResult result, IndexDefinition indexDefinition, IIndexActions actions, bool retryOnError)
+        {
+            try
+            {
+                if (_options.RemoveAnalyzers)
+                {
+                    foreach (var indexDefinitionField in indexDefinition.Fields)
+                        indexDefinitionField.Value.Analyzer = null;
+                }
+
+                actions.WriteIndex(indexDefinition);
+            }
+            catch (Exception e)
+            {
+                var exceptionMessage = e.Message;
+                if (exceptionMessage.Contains("No overload for method 'LoadDocument' takes 1 arguments"))
+                {
+                    exceptionMessage = $"{e.Message}" + Environment.NewLine +
+                            "LoadDocument requires a generic argument for its usage (the type of the loaded document)" + Environment.NewLine +
```

Comment:
Shouldn't this be: LoadDocument requires a second argument which is a collection name of the loaded document (or something like that) ?